source/gent/tier1_hortense.rst

```diff
@@ -60,33 +52,103 @@ Shared infrastructure:
 Getting access
 --------------
 
+.. _hortense_access_policy:
+
+Access policy
+*************
+
 **The Hortense VSC Tier-1 cluster can only be used by people with an active Tier-1 compute project.**
 
 See https://www.vscentrum.be/compute for more information on requesting access.
 
-To access the Tier-1 Hortense cluster, use SSH to connect to the dedicated login node with your VSC account:
+.. _hortense_login_nodes:
+
+Login nodes (SSH)
+*****************
+
+You can use SSH to connect to the login nodes of the Tier-1 Hortense cluster with your VSC account:
 
 * from the public internet, use ``tier1.hpc.ugent.be``
 * from within the VSC network, use ``tier1.gent.vsc``
```

Comment:
You can also make an extra ssh key to use from Tier2 as well (besides the ssh agent)

Reply:
We should document agent forwarding, and creating an extra key in your home dir is an alternative, but that's definitely outside of scope for this document, since it's very generic (also applies to hopping between Tier-2 login nodes, etc.).
We refer to the `acccess_data_transfer` section here, so those aspects should be documented there, not here.